apps/sql-server/Dockerfile

```diff
@@ -23,28 +23,39 @@ RUN echo "listen_addresses = '*'" >> /etc/postgresql/${POSTGRES_VERSION}/main/po
 RUN echo "host all all 0.0.0.0/0 md5" >> /etc/postgresql/17/main/pg_hba.conf
 
 # Postgres/Multicorn insists on using the system Python, so we need to disable the virtual environment
+# Store current VIRTUAL_ENV and PATH values
+ENV OLD_VIRTUAL_ENV="${VIRTUAL_ENV}"
+ENV OLD_PATH="${PATH}"
+ENV OLD_PYTHONPATH="${PYTHONPATH}"
+
+# Disable virtual environment
 ENV VIRTUAL_ENV=
 ENV PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+ENV PYTHONPATH=""
 
 # Install multicorn2 Python module into system Python
 RUN /usr/bin/python3 -m pip install --no-cache-dir "git+https://github.com/pgsql-io/multicorn2.git"
 
-# Build and install multicorn2 Postgres extension
-RUN git clone https://github.com/pgsql-io/multicorn2.git /multicorn2 \
+# Build and install multicorn2 Postgres extension 
+RUN git clone  --single-branch --branch main https://github.com/pgsql-io/multicorn2.git /multicorn2 \
  && cd /multicorn2 \
  && make PYTHON=/usr/bin/python3 \
  && make install
 
-RUN pip install --no-cache-dir aperturedb dotenv
+RUN pip install --no-cache-dir aperturedb dotenv numpy pydantic
+
+# Install dependencies for embeddings
+RUN pip install --no-cache-dir -r /app/embeddings/requirements_cpu.txt
```

Comment:
Does the index url determine install type for subsequent pip installs? If so, perhaps do install aperturedb after this, or it will pull in torch and all, with default as gpu